game/mushcnf.dst

```diff
@@ -1017,3 +1017,10 @@ include alias.cnf
 ###
 include restrict.cnf
 
+### JSON Configuration Options
+# JSON Safety by default is active. This means that during unescaping of any 
+# json functions, it does not unescape Markup sequences. This is because 
+# it is possible for a player to cause ansi to bleed etc. We recommend to 
+# keep this off unless you know what you are doing. If you set this to 1, 
+# we recommend making all json functions Roy or Wiz based using @function/restrict
+json_unsafe_unescape 0
```

Comment:
I have no idea what this option actually does from this description

Reply:
New description added. Is it clearer now?